airflow/Dockerfile

```diff
@@ -48,6 +48,8 @@ RUN python${PYTHON} -m venv --system-site-packages /stackable/app && \
     pip install --no-cache-dir apache-airflow[${AIRFLOW_EXTRAS}]==${PRODUCT} --constraint /tmp/constraints.txt && \
     # Needed for pandas S3 integration to e.g. write and read csv and parquet files to/from S3
     pip install --no-cache-dir s3fs cyclonedx-bom && \
+    # Needed for OIDC
+    pip install --no-cache-dir --upgrade Flask_OIDC==2.2.0 Flask-OpenID==1.3.1 && \
```

Comment:
I'm a pip noob, is the --upgrade needed to upgrade pip itself or to say "install this lib in this version, update it in case you already have an older version installed"?

Reply:
you were right! done with https://github.com/stackabletech/docker-images/pull/891/commits/8662bc268febbb2722bc47a85f744cf9162cd86e

Reply:
Thanks!